docs/quickstart.md

```diff
@@ -54,10 +54,8 @@ The resulting benchmark executions will reside in this namespace.
 
 We will be using [kustomize](https://kustomize.io/) to render the Custom Resource from the [github repository](https://github.com/xridge/kubestone/tree/master/config/samples/fio/).
 
-Kustomize takes a [base yaml](https://github.com/xridge/kubestone/blob/master/config/samples/fio/base/fio_cr.yaml) and patches with an [overlay file](https://github.com/xridge/kubestone/blob/master/config/samples/fio/overlays/pvc/patch.yaml) to render the final yaml file, which describes the benchmark.
```

Comment:
I'm sorry to see this info go away. Would you mind adding it back as part of an other example?

Reply:
readded